### PR TITLE
dts: add support for FMC signals

### PR DESCRIPTION
**Pull request by gmarull**
Add Flexible Memory Controller (FMC) signals. Only available on some non-F1 series.

### CODE REVIEW

**Review by erwango — APPROVED** [review]

**Comment by lochej** [comment]
You should probably rebase this PR so it fixes your buildkite issue https://github.com/zephyrproject-rtos/zephyr/pull/29686 😄

**Comment by erwango** [comment]
@lochej woul you mind approve this one also, if this is fine with you ?
We'd need a second approver to ask for merge

**Review by lochej — APPROVED** [review]

**Review by ABOSTM — APPROVED** [review]
LGTM

**Pull request merged**
